DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs

```diff
@@ -60,44 +59,66 @@ public class RepositoryController : BaseApiController
         [Route(RouteConstants.GetCollaboratorsForRepository)]
         public async Task<IHttpActionResult> GetCollaboratorsForRepo(string repoName)
         {
-            string token = User.Identity.GetExternalAccessToken();
+            var claims = User.Identity as ClaimsIdentity;
+            string token = claims.FindFirstValue("ExternalAccessToken");
             string userName = User.Identity.GetUserName();
 
-            var gitHubCollaborators = await _gitHubService.GetCollaboratorsForRepo(userName, repoName, token);
+            var gitHubCollaborators = await _guGithubService.GetCollaboratorsForRepo(userName, repoName, token);
 
             return Ok(gitHubCollaborators);
         }
 
         [Authorize]
         [Route("api/repos/current")]
         [HttpPatch]
+        //TODO: use DTO
         public async Task<IHttpActionResult> UpdateProject(Repository repo)
         {
             string currentUserId = User.Identity.GetUserId();
             User user = await _userManager.Users.FirstOrDefaultAsync(x => x.Id == currentUserId);
             if (user == null)
                 return NotFound();
 
-            var repository = _securityContext.UserRepository.FirstOrDefault(r => r.RepositoryId == repo.Id && r.UserId == user.Id);
-            if (repository == null)
+            var userRepositoryRole = _securityContext.UserRepository.FirstOrDefault(r => r.RepositoryId == repo.Id && r.UserId == user.Id);
+            if (userRepositoryRole == null)
             {
                 ModelState.AddModelError("repo", string.Format("user with id '{0}' does not have a repository with id '{1}'", user.Id, repo.Id));
                 return BadRequest(ModelState);
             }
 
-            var claimsIdentity = await user.GenerateUserIdentityAsync(_userManager, DefaultAuthenticationTypes.ApplicationCookie);
-            var existingClaim = claimsIdentity.Claims.FirstOrDefault(c => c.Type == "CurrentProjectId");
-            if (existingClaim != null)
-                _userManager.RemoveClaim(user.Id, existingClaim);
+            var claimsIdentity = User.Identity as ClaimsIdentity;
+            string[] claimTypes = { "CurrentProjectId", "CurrentProjectName" };
```

Comment:
because it was already fixed in https://github.com/ITsvetkoFF/Kv-013/blob/feature/activity-external-backend/DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs